Jenkinsfile

```diff
@@ -96,11 +93,10 @@ pipeline {
                 timeout(time: 3, unit: 'HOURS')
             }
             steps {
-                // TODO: Maven downgraded to work around a linux build issue. Falling back to system java to work around a linux build issue. re-investigate upgrading later
                 withMaven(maven: 'maven-latest', globalMavenSettingsConfig: 'default-global-settings', mavenSettingsConfig: 'codice-maven-settings', mavenOpts: '${LARGE_MVN_OPTS} ${LINUX_MVN_RANDOM}') {
                     sh '''
                         unset JAVA_TOOL_OPTIONS
-                        mvn install -B -pl $ITESTS -nsu $DISABLE_DOWNLOAD_PROGRESS_OPTS
+                        mvn install -B -pl $ITESTS,$ITCORE -nsu $DISABLE_DOWNLOAD_PROGRESS_OPTS
```

Comment:
Can we put core tests in their own stage? I think this will help us view its stability and potentially replace the itests only stage in the future. @pklinef thoughts?

Reply:
That would be very helpful for the reasons you gave as long as it does not add too much overhead.